UMLEditor/UMLEditor/Views/AddRelationshipPanel.axaml

```diff
@@ -10,27 +10,22 @@
         
         <Label
             Margin="0 10 0 5"
-            Content = "Enter source class name:">
+            Content = "Select source class name:">
         </Label>
         
-        <TextBox
-            Watermark="Source class goes here"
-            Name="SourceClass">
-        </TextBox>
+        <ComboBox SelectedIndex="0" Name="SourceClassSelector" HorizontalAlignment="Stretch" />
         
         <Label
             Margin="0 10 0 5"
-            Content = "Enter destination class name:">
+            Content = "Select destination class name:">
         </Label>
         
-        <TextBox
-            Watermark="Destination class goes here"
-            Name="DestinationClass">
-        </TextBox>
+        <ComboBox SelectedIndex="0" Name="DestClassSelector" HorizontalAlignment="Stretch" />
         
         <Label
             Margin="0 10 0 5"
-            Content = "Enter relationship type:">
+            Content = "Enter relationship type:"
```

Comment:
Could we change this to match the other two prompts? "Select relationship type:"